### DISCUSSION TITLE
[即时战略]帝国时代3：亚洲王朝

### DISCUSSION THREAD

**Discussion by 1409130272** [Gamuxorg/bbs#337] [投递和请求游戏]
游戏名称：帝国时代3：亚洲王朝
游戏名称(英文)：Age of Empires III The Asian Dynasties
下载地址 :（如果有的话）：
[](http://www.pc0359.cn/downinfo/65243.html)

**Comment by 1409130272** [comment]
下载地址：http://www.pc0359.cn/downinfo/65243.html

**Comment by 1409130272** [comment]
CrossOver 官网上是有这个游戏的...应该能够实现...
![微信图片_20190804194800](https://user-images.githubusercontent.com/11330048/62423204-f1675c80-b6f0-11e9-8c20-3bb93ea69552.png)

**Comment by wangling12** [comment]
打包wine游戏很麻烦，告诉你运行方法吧
以你给的资源为例，安装最新的wine和winetricks
进入解压完的文件夹

初始化wine配置：
`WINEPREFIX=$(pwd)/wine winetricks dlls l3codecx`
`WINEPREFIX=$(pwd)/wine winetricks dlls quartz `
`WINEPREFIX=$(pwd)/wine winetricks dlls mfc42`
`WINEPREFIX=$(pwd)/wine winetricks dlls d3dx9 `
`WINEPREFIX=$(pwd)/wine winetricks settings winxp `
`WINEPREFIX=$(pwd)/wine winetricks settings vd=1024x768`

然后你的目录大概长这样：

> 10101817
> ├── Age of Empires III
> ├── config.ini
> ├── Setup.exe
> └── wine

安装：
`WINEPREFIX=$(pwd)/wine wine Setup.exe`

我将游戏安装到了10101817/ 
即我的文件结构是：

> 10101817
> ├── 10101817
> │   └── Age3xLauncher.exe
> ├── Age of Empires III
> ├── config.ini
> ├── Setup.exe
> └── wine

运行游戏：
`WINEPREFIX=$(pwd)/wine wine 10101817/Age3xLauncher.exe`

**Comment by 1409130272** [comment]
好的，谢谢大神 




------------------ 原始邮件 ------------------
发件人: "wangling12"<notifications@github.com>;
发送时间: 2019年8月4日(星期天) 晚上9:29
收件人: "Gamuxorg/bbs"<bbs@noreply.github.com>;
抄送: "未来"<1409130272@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [Gamuxorg/bbs] [即时战略]帝国时代3：亚洲王朝 (#59)




打包wine游戏很麻烦，告诉你运行方法吧
 以你给的资源为例，安装最新的wine和winetricks
 进入解压完的文件夹
 
初始化wine配置：
 WINEPREFIX=$(pwd)/wine wine
 WINEPREFIX=$(pwd)/wine winetricks dlls l3codecx
 WINEPREFIX=$(pwd)/wine winetricks dlls quartz 
 WINEPREFIX=$(pwd)/wine winetricks dlls mfc42
 WINEPREFIX=$(pwd)/wine winetricks dlls d3dx9 
 WINEPREFIX=$(pwd)/wine winetricks settings winxp 
 WINEPREFIX=$(pwd)/wine winetricks settings vd=1024x768
 
然后你的目录大概长这样：
  
10101817
 ├── Age of Empires III
 ├── config.ini
 ├── Setup.exe
 └── wine
  
安装：
 WINEPREFIX=$(pwd)/wine wine Setup.exe
 
我将游戏安装到了10101817/
 即我的文件结构是：
  
10101817
 ├── 10101817
 │   └── Age3xLauncher.exe
 ├── Age of Empires III
 ├── config.ini
 ├── Setup.exe
 └── wine
  
运行游戏：
 WINEPREFIX=$(pwd)/wine wine 10101817/Age3xLauncher.exe
